certauth.go

```diff
@@ -61,10 +62,46 @@ type Options struct {
 
 // Auth is an instance of the middleware
 type Auth struct {
-	opt            Options
-	authErrHandler http.Handler
+	opt          Options /* **DEPRECATED** */
+	checkers     [][]AuthorizationChecker
```

Comment:
Now that I'm thinking about it, it may be worthwhile to put a comment here explaining that this is a list-of-lists in this form:

`OR(AND(x for x in y) for y in checkers)`